manifests/api/service.pp

```diff
@@ -31,11 +31,13 @@
       }
     }
 
-    service { 'sensu-api':
-      ensure     => $ensure,
-      enable     => $enable,
-      hasrestart => $hasrestart,
-      subscribe  => [ Class['sensu::package'], Class['sensu::api::config'], Class['sensu::redis::config'] ],
+    if $::osfamily != 'windows' {
+      service { 'sensu-api':
```

Comment:
As far as I know and per the [documentation](https://sensuapp.org/docs/latest/windows-service-configuration) I believe only the client is supported on windows.